src/site/content/en/blog/appcache-removal/index.md

```diff
@@ -0,0 +1,211 @@
+    </td>
+    <td>Chrome 85 (<a href="https://chromiumdash.appspot.com/schedule">estimated August 2020</a>)
+    </td>
+    </tr>
+    <tr>
+    <td>Complete removal from secure contexts for everyone, with completion of origin trial
+    </td>
+    <td>Chrome 90 (<a href="https://chromiumdash.appspot.com/schedule">estimated April 2021</a>)
+    </td>
+    </tr>
+  </table>
+</div>
+
+{% Aside %}
+This timeline applies to Chrome on **all platforms other than iOS**. There is also an adjusted timeline for AppCache used within an Android [WebView](https://developer.android.com/reference/android/webkit/WebView). For more info, see [The cross-platform story](#the-cross-platform-story) later in this post.
+{% endAside %}
+
+## Origin trial
+
+The timeline lists two upcoming milestones for removal. Beginning with Chrome 85, AppCache will no longer be available in Chrome by default. Developers who require additional time to migrate off of AppCache can [sign up](https://developers.chrome.com/origintrials/#/register_trial/1776670052997660673) for a "reverse" [origin trial](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md) to extend the availability of AppCache for their web apps. The origin trial will start in Chrome 84 (in advance of the default removal in Chrome 85), and will be active through Chrome 89. Starting with Chrome 90, AppCache will be fully removed for everyone, even those who had signed up for the origin trial.
+
+{% Aside %}
+Why are we calling this a "reverse" origin trial? Normally, an origin trial allows developers to opt-in to early access to new functionality before it has shipped by default in Chrome. In this case, we're allowing developers to opt-in to using legacy technology even after it's been removed from Chrome.
+{% endAside %}
+
+To participate in the "reverse" origin trial:
+
+- [Request a token](https://developers.chrome.com/origintrials/#/register_trial/1776670052997660673) for your origin.
+
+- Add the token to your HTML pages. There are [two ways](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md#how-do-i-enable-an-experimental-feature-on-my-origin) to do that:
+
+  - Add an `origin-trial` `<meta>` tag to the head of each page. For example: `<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">`
+
+  - Alternatively, configure your server to return responses containing the `Origin-Trial` HTTP header. The resulting response header should look something like: `Origin-Trial: TOKEN_GOES_HERE`
+
+- Add the same token to your AppCache manifests. This is done via a new field in your manifest, with the format:
+
+```text
+ORIGIN-TRIAL:
+TOKEN_GOES_HERE
+```
+
+(There needs to be a new line between `ORIGIN-TRIAL:` and your token.)
+
+{% Aside %}
+The token for a manifest **must** be in an `ORIGIN-TRIAL:` field of the manifest itself. Unlike an HTML page's token, it can't be provided via an HTTP header.
```

Comment:
```suggestion
The token for a manifest **must** be in an `ORIGIN-TRIAL` field of the manifest itself. Unlike an HTML page's token, it can't be provided via an HTTP header.
```